lib/xeroizer/record/xml_helper.rb

```diff
@@ -107,6 +107,7 @@ def xml_value_from_field(b, field, value)
                 real_value = case value
                   when Date         then value.strftime("%Y-%m-%d")
                   when Time         then value.utc.strftime("%Y-%m-%d")
+                  when String       then Date.parse(value).strftime("%Y-%m-%d")
```

Comment:
I agree with both CloCkWeRX and waynerobinson that error handling here is a bastard, so I'll emend this pull request to throw an error if a string is provided instead. Good plan.